cpp/core/shuffle/RoundRobinPartitioner.h

```diff
@@ -23,8 +23,8 @@ namespace gluten {
 
 class RoundRobinPartitioner final : public Partitioner {
  public:
-  RoundRobinPartitioner(int32_t numPartitions, int32_t startPartitionId)
-      : Partitioner(numPartitions, false), pidSelection_(startPartitionId % numPartitions) {}
+  RoundRobinPartitioner(int32_t numPartitions, int32_t startPartitionId, bool hasPid)
```

Comment:
`hasPid` should be `sortBeforeRR`, if we need this parameter.

Reply:
addressed.